pkgs/applications/graphics/pqiv/default.nix

```diff
@@ -12,7 +33,13 @@ stdenv.mkDerivation (rec {
   };
 
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ getopt which gtk3 ];
+  buildInputs = [ getopt which gtk ]
+    ++ stdenv.lib.optional supportArchives libarchive
+    ++ stdenv.lib.optional supportMisc imagemagick
+    ++ stdenv.lib.optional supportPDF poppler
+    ++ stdenv.lib.optional supportPS libspectre
+    ++ stdenv.lib.optional supportVideo ffmpeg
+    ++ stdenv.lib.optional supportWebP libwebp;
```

Comment:
Do we really options for that? 
People can do `pqiv.override { libarchive = null; }` to disable dependencies.

Reply:
Also

```
qiv = callPackage ../applications/graphics/pqiv {
  enablePDF = config.pqiv.enablePDF;
};
```

is a better approach, since it does not pull options from global namespace.